evalml/utils/gen_utils.py

```diff
@@ -207,7 +209,7 @@ def get_importable_subclasses(base_class, used_in_automl=True):
         try:
             cls()
             classes.append(cls)
-        except (ImportError, MissingComponentError, TypeError):
+        except (ImportError, MissingComponentError, TypeError, ValueError):
```

Comment:
What about Prophet would raise a ValueError when trying to instantiate the class? I think `ValueErrors` are pretty broad so I'm wondering if we can make the exception "narrower".

Reply:
Hmm I think this was a different issue that's no longer applicable